app/main.py

```diff
@@ -1,3 +1,55 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return Distance(self.km)
```

Comment:
in this method you modify `self.km`, but you shouldn't do this. You have to return different `Distance` object and pass to it new value, that is calculated by `self.km + other.km` or `self.km + other`